deeds-dapp-service/src/main/java/io/meeds/deeds/service/BlockchainService.java

```diff
@@ -101,4 +117,33 @@ public short getDeedCityIndex(long nftId) throws ObjectNotFoundException {
     }
   }
 
+  public BigDecimal totalSupply() {
+    try {
+      BigInteger totalsupply = ethereumToken.totalSupply().send();
+      return new BigDecimal(totalsupply).divide(BigDecimal.valueOf(10).pow(18));
+    } catch (Exception e) {
+      throw new IllegalStateException("Error retrieving information 'totalSupply' from Blockchain", e);
```

Comment:
can we add the name/symbol (at least address) of the token for clarity?

Reply:
"Error retrieving information 'totalSupply' for Meeds Token from Blockchain"
WDYT @plamarque  ?